resources/index.ts

```diff
@@ -96,6 +99,7 @@ export const supportedDAOs: ISupportedDAOs = {
     dark: starknet.dark,
     light: starknet.light,
     ABI: null,
+    TOKENABI: starknet.TOKENABI,
```

Comment:
What is the difference between ABI and TOKENABI? Why doesn't all DAOs have both?

Reply:
ABI is the delegation contract ABI.
TOKENABI is the token contract ABI.

I'll change the ABI name to DELEGATIONABI, to be less confuse.

Also, if all DAOs have both but doesn't use both, it's just a duplicated code/file (weighing the repository and code).